test/inductor/test_torchinductor_opinfo.py

```diff
@@ -322,6 +322,7 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
     ("cauchy", "cuda"): {"reference_in_float": True},
     ("cummax", "cuda", f16): {"atol": 5e-4, "rtol": 0.002},
     ("cumprod", "cuda"): {"reference_in_float": True, "atol": 7e-5, "rtol": 0.002},
+    ("logcumsumexp", "cuda"): {"atol": 8e-4, "rtol": 0.001},
```

Comment:
This tolerance is rather high but I see it's only relevant to the backward case. Do you think we could ad a `grad_atol` and `grad_rtol` to the check function so we don't have to be so loose for the forward case?